### ISSUE TITLE
Having trouble installing Instapy for the first time

### ISSUE THREAD

**Issue by anubisfkr** [InstaPy/InstaPy#4585]
Hi, hi, so I need some help. I want to install instapy, but I'm having trouble. I downloaded Python 3.7.3, and I tried opening the video walk through for Windows, but... I wasn't able to follow it. Not sure if it's outdated or if I'm that clueless. I'm still learning how to program, so it's probably the latter. I also have my own quickstarts that I wrote, and I want to use those, so how would I go about imputing those? Can someone please help me? I'm willing to give out my discord or email to chat about the issue.

**Comment by Herrox** [comment]
Hello,

Feel free to join the discord server, it may be easier for help. Regarding your problem, it would be great if you can provide more details about it : where are you stuck? Do you have eny error message? Have you tried to watch and install again?

**Comment by anubisfkr** [comment]
I did join! I got advice there, they said I should try using the GUI version. I don't get an error message. So, I download the Instapy zip file, extract it, and then double click on a python file, but nothing happens. The python box comes open for a split second and then disappears. The video starts off with extracting the files, and then when he opens them, there is an install file. There isn't one like that for me. So, I'm not really sure what I'm doing wrong.

**Comment by simonel** [comment]
> 
> 
> Hi, hi, so I need some help...

Hello Jae! After extraction, you should have a 'run' and an 'installation' folder, which contains the Windows folder and install file. After installation, you should open the quickstart (python file), update it with your customized code, save it, then double-click it to run. If your script is correct and you have installed all of the proper dependencies (using the command line where appropriate), you should see your custom script in action via your Chrome web browser. If the Python window closes immediately upon opening AND provided that you have properly installed all components, then you can evaluate the error(s) in your quickstart.py file using the Wing IDE editor:

right-click on the quickstart.py file > edit with WING IDE Personal 6.1 > click 'python shell' (bottom right-hand window pane) > Source (top tab) > Evaluate File in Python Shell.

**Comment by anubisfkr** [comment]
Hey @simonel ! Sorry for the late response. I thought I would get emails whenever someone commented... clearly not, haha. Even after extracting it, what I get is a list of these files.
![image](https://user-images.githubusercontent.com/52257130/61002074-721b7e80-a326-11e9-99a1-d19fe10def4d.png)
However, I did try what you said about editing the file in IDLE, and I am able to, but here's a more mundane, silly question. What's the exact command to run or open a file in Python? I looked at help, but there's so much information, I'm not sure where to start ;;

And thanks again for all your help, I seriously appreciate it.
